src/normalize/quotation.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 OpenStax Poland
+// Licensed under the MIT license. See LICENSE file in the project root for
+// full license text.
+
+import { Editor, Node, NodeEntry, Range, Transforms } from 'slate'
+import { List } from 'slate-lists'
+
+import { Paragraph, Quotation, Title } from '../interfaces'
+import { normalizeOrderedChildren } from './util'
+
+/**
+ * Normalize an quotation
+ *
+ * Return true if entry was normalized and false otherwise.
+ */
+export default function normalizeQuotation(editor: Editor, entry: NodeEntry): boolean {
+    const [node, path] = entry
+
+    if (Quotation.isQuotation(node)) {
+        // A quotation must not be empty.
+        if (Editor.isEmpty(editor, node)) {
+            Transforms.removeNodes(editor, { at: path })
+            return true
+        }
+
+        // Unwrap nested quotation if it is not selected
+        // and it's the only child of another quotation.
+        if (node.children.length === 1
+            && Quotation.isQuotation(node.children[0])
+            && (editor.selection == null || !Range.includes(editor.selection, path))
+        ) {
```

Comment:
Last nitpick: please keep style of `if` conditions consistent.

Reply:
```js
if (node.children.length === 1
&& Quotation.isQuotation(node.children[0])
&& (editor.selection == null || !Range.includes(editor.selection, path))
) {
```
?

Reply:
```typescript
if (node.children.length === 1
&& Quotation.isQuotation(node.children[0])
&& (editor.selection == null || !Range.includes(editor.selection, path))) {
```